client/debian/control

```diff
@@ -22,7 +22,13 @@ Rules-Requires-Root: no
 Package: librust-hwlib-dev
 Architecture: amd64
 Multi-Arch: foreign
-Depends: libssl-dev, pkgconf, python3, ${misc:Depends}, ${shlibs:Depends}
+Depends: kmod,
```

Comment:
Does the devel package actually need those? Are they really invoked from tests (not mocked)?

Reply:
The library uses these tools to collect OS information

Reply:
Yes but how does the developer package need them? Those should be on the non-devel package.

Reply:
if you want to run the code that sends the certification status request, you need these packages

Reply:
Right but who needs the -devel package? Someone bundling the library in their project (like hwctl). The devel package does not actually need them, right? You don't end up with a runtime dependency on -dev package.